Demo/build.gradle

```diff
@@ -63,7 +63,6 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.4.0-alpha03'
     implementation 'androidx.cardview:cardview:1.0.0'
 
-    implementation 'com.google.android.gms:play-services-wallet:16.0.1'
```

Comment:
Is this worthy of a CHANGELOG entry or not something that merchants would notice?

Reply:
They shouldn't notice it - we were duplicate including this dependency (already bundled with the core Google Play module) so it was already resolving to the higher version from core